docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -72,9 +81,9 @@ const frame = new MutableDataFrame({
 
 ## Add labels to your logs
```

Comment:
Are these not optional enhancements?

Reply:
Yes. I removed "(Optional)" because the title of this section changed to "## Enhance your logs data source plugin with optional features" and I avoided redundancy. If this is confusing, let's change it though.

Reply:
> because the title of this section changed to "## Enhance your logs data source plugin with optional features" and I avoided redundancy

That I get, but the description for the new **section** has "Add visualization type hints, labels, and other optional features to logs." but then as shown above, the labels content is not _within_ this section, it's its own section.

Reply:
Issue is fixed in latest commit.